proxy/src/project.rs

```diff
@@ -214,12 +215,11 @@ where
 
     /// Set up the PATH env variable used for running the checkout.
     fn default_bin_path() -> Result<ffi::OsString, error::Error> {
```

Comment:
Maybe this could go in `config` too?

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/769/commits/0bcf74272d1853758f9d9b0296eaad4304423b27